### DISCUSSION TITLE
Scroll bars

### DISCUSSION THREAD

**Discussion by zs-dima** [mikekazakov/nimble-commander#384] [Ideas]
Specially for panels in Full mode. 
1. Could be nice to be able to select color for scroll bars placeholders.
2. Could be nice to have an option to hide unneeded scrollbars.
For example horizontal scroll bars placeholders are visible even if it is no horizontal scroll bars on both panels.

**Comment by mikekazakov** [comment]
Nimble Commander uses system-defined behaviour and presentation of the scrollbars.
Customisations are possible, but it's a work somebody has to do - PRs are welcome.

**Reply by zs-dima** [reply]
@mikekazakov strange, Finder don't display horizontal scroll bar placeholder when it's nothing to scroll horizontally. Or maybe I missed some system settings.

**Reply by mikekazakov** [reply]
I've looked into it briefly. Seems like Finder is breaking MacOS's own UI conventions here and forcefully hides the scrollbars in such cases. Well, NC can do the same I guess.
Though just of curiosity - why do you have this option turned on in the first place?
